srcpkgs/haskell-language-server/template

```diff
@@ -0,0 +1,33 @@
+# Template file for 'haskell-language-server'
+pkgname=haskell-language-server
+version=1.0.0
+revision=1
+archs="x86_64"
+build_style="haskell-stack"
+make_build_args="--stack-yaml stack-8.8.4.yaml"
+makedepends="ncurses-devel ncurses-libtinfo-devel icu-devel zlib-devel"
+short_desc="Haskell Language Server: Integration of ghcide and haskell-ide-engine"
+maintainer="Wayne Van Son <waynevanson@gmail.com>"
+license="Apache-2.0"
+homepage="https://github.com/haskell/haskell-language-server"
+distfiles="https://github.com/haskell/haskell-language-server/archive/${version}.tar.gz"
+checksum="14e28d6621d029f027fae44bc4a4ef62c869dab24ff01b88a2e51e6679cbff6c"
+nopie_files="
+/usr/bin/haskell-language-server
+/usr/bin/haskell-language-server-wrapper
+/usr/bin/ghcide-bench
+/usr/bin/ghcide
+/usr/bin/ghcide-test-preprocessor
+/usr/bin/test-server
+"
+nocross="Haskell package 'Network' builds the hosts architecture, not the target's"
```

Comment:
build_style="haskell-stack" can't cross compile in general.

Reply:
Okay thanks for letting me know.
Should I remove the line or comment it out?

Reply:
Please leave and possibly change to nocross="Can't yet cross compile Haskell".